spec/check_graphite_spec.rb

```diff
@@ -3,6 +3,15 @@
 
 require "check_graphite"
 
+RSpec.configure do |config|
+  config.expect_with :rspec do |c|
+    c.syntax = [:should, :expect]
+  end
+  config.mock_with :rspec do |c|
+    c.syntax = [:should, :expect]
+  end
```

Comment:
It doesn't look like the tests use `expect`, so to avoid someone making an inconsistent choice, maybe not even enable those?